fixtures/dom/src/components/Header.js

```diff
@@ -43,6 +52,15 @@ class Header extends React.Component {
           </span>
 
           <div className="header-controls">
+            <input
+              id="react_production"
+              type="checkbox"
+              checked={this.state.production}
+              onChange={this.handleProductionChange}
+            />
+            <label htmlFor="react_production">
+              <span className="header__label">Production</span>
```

Comment:
Could you eliminate this extra span by adding the class name to the label?

Reply:
Yes, good idea.